h/browser/chrome/manifest.json

```diff
@@ -20,7 +20,11 @@
     "<all_urls>",
 
     "contentSettings",
-    "tabs"
+    "tabs",
+
+    "fileBrowserHandler",
+    "webRequest", "webRequestBlocking",
+    "storage"
```

Comment:
Yeah. Left it there as I plan to use it for the PDF URL to media type "recording." Possibly going to upgrade TabState also...later...to use it. We'll see.